code/modules/surgery/organs/internal/ears/_ears.dm

```diff
@@ -67,12 +67,13 @@
 	icon_state = "kitty"
 	visual = TRUE
 	damage_multiplier = 2
+	var/ear_to_use = "Cat"
```

Comment:
I haven't gone through every var and said "you should doc these" (although you probably should) but this one seems important because it's not intuitive that you'd need to specify that cat ears are not rabbit ears, given that it says that they are cat ears